### PR TITLE
Added logo and theming to docs, also update to gitignore

### PR DESCRIPTION
**Pull request by ehsteve**
Added mac files to gitignore

### CODE REVIEW

**Review by dbarrous — APPROVED** [review]
LGTM!

File: docs/conf.py [hunk 1]
```diff
@@ -12,8 +12,8 @@
 # -- Project information -----------------------------------------------------
 
 project = "hermes_eea"
-copyright = "2021, Jane Doe"
-author = "Jane Doe"
+copyright = "No rights reservers"
```
Comment (rstrub) [inline docs/conf.py:15]: Is reservers a typo

Comment (ehsteve) [reply docs/conf.py:15]: It is a typo. Thanks for catching that. I actually just read through an NPR that suggests that we should add a copyright notice even though as a work of the US government we can't claim copyright protection.

**Comment by ehsteve** [comment]
I will make this change in `hermes_instrument` to propagate to all instrument repos.

**Pull request closed without merging**